test/object-store/realm.cpp

```diff
@@ -592,7 +592,7 @@ TEST_CASE("SharedRealm: get_shared_realm()") {
         auto realm = Realm::get_shared_realm(config);
         realm->read_group();
         auto frozen_realm = realm->freeze();
-        REQUIRE(frozen_realm->schema() == schema);
+        REQUIRE(frozen_realm->is_frozen());
```

Comment:
This is specifically a repro case for a bug that we had and is an important thing to be checking.

Reply:
I think the same as stated above applies, Is it safe to return the OS schema?

Reply:
this is now outdated.